posts/MarkoB/2024-04-10-hibernate-search-maintenance-releases.adoc

```diff
@@ -0,0 +1,117 @@
+= Hibernate Search 7.1.1.Final/7.0.1.Final/6.2.4.Final are out!
+Marko Bekhta
+:awestruct-tags: [ "Hibernate Search", "Lucene", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+:hsearch-doc-url-prefix: https://docs.jboss.org/hibernate/search/7.1/reference/en-US/html_single/
+:hsearch-getting-started-orm-url-prefix: https://docs.jboss.org/hibernate/search/7.1/getting-started/orm/en-US/html_single/
+:hsearch-getting-started-standalone-url-prefix: https://docs.jboss.org/hibernate/search/7.1/getting-started/standalone/en-US/html_single/
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-version-family-71: 7.1
+:hsearch-version-family-70: 7.0
+:hsearch-version-family-62: 6.2
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id-71: 32269
+:hsearch-jira-version-id-70: 32226
+:hsearch-jira-version-id-62: 32237
+---
+
+We are pleased to announce the releases of Hibernate Search 7.1.1.Final, 7.0.1.Final and 6.2.4.Final.
+
+While these releases are mainly to address some minor documentation discrepancies,
+they also bring compatibility with newer versions of Elasticsearch and OpenSearch
+and a few bugfixes.
+
++++<!-- more -->+++
+
+== Hibernate Search 7.1.1.Final
+
+=== Dependency upgrades
```

Comment:
No mention of ORM 6.5?

Reply:
mmm... we only upgraded to 6.5 in the main branch, and since 6.5 is just a CR for now ... I wasn't sure we should mention anything 🙈 

```
[[orm-version-7-1]]
Hibernate ORM::
While Hibernate Search 7.1 still depends on Hibernate ORM 6.4, it is being tested and is compatible with the upcoming Hibernate ORM 6.5.
```

?

Reply:
> ```
> [[orm-version-7-1]]
> Hibernate ORM::
> While Hibernate Search 7.1 still depends on Hibernate ORM 6.4, it is being tested and is compatible with the upcoming Hibernate ORM 6.5.
> ```

:+1: